base/bucket_n1ql.go

```diff
@@ -48,7 +48,11 @@ func (bucket *CouchbaseBucketGoCB) Query(statement string, params interface{}, c
 
 	waitTime := 10 * time.Millisecond
 	for i := 1; i <= MaxQueryRetries; i++ {
+
+		Tracef(KeyQuery, "Executing N1QL query: %v", UD(n1qlQuery))
 		queryResults, queryErr := bucket.ExecuteN1qlQuery(n1qlQuery, params)
+		Tracef(KeyQuery, "Done executing N1QL query: %v.  Error: %v", UD(n1qlQuery), queryErr)
```

Comment:
Can we remove this logging (or at least only log in the queryErr != nil case?)

Reply:
Removed